tajo-core/src/test/resources/results/TestJoinBroadcast/testCrossJoinWithAsterisk1.result

```diff
@@ -1,8 +1,8 @@
 r_regionkey,r_name,r_comment,c_custkey,c_name,c_address,c_nationkey,c_phone,c_acctbal,c_mktsegment,c_comment
```

Comment:
@jihoonson recommend to add an order by clause to queries if necessary rather than just changing the order of rows of query results. I agree with this opinion. 
For example, you can add 'c_custkey' column in order by clause in "testCrossJoinWithAsterisk1.sql".